template/scripts/deploy.js

```diff
@@ -1,17 +1,17 @@
+const { coin } = require('@cosmjs/stargate');
+
 async function main () {
   const [alice, bob] = await getSigners()
+  const fund = [coin(321, "acudos")];
+
   const contract = await getContractFactory('alpha', bob)
   
   const MSG_INIT = { count: 13 }
-  // deploying as bob
-  const deploy = await contract.deploy(MSG_INIT)
+  const deploy = await contract.deploy(MSG_INIT, fund)
   const contractAddress = deploy.initTx.contractAddress
   console.log(`Contract deployed at: ${contractAddress}`)
 }
 
-main()
```

Comment:
I think we should keep this pattern so the user can use async/await everywhere and properly handle errors. We can even put a comment about this.